### ISSUE TITLE
[BUG] - Target [Doctrine\Persistence\ManagerRegistry] is not instantiable

### ISSUE THREAD

**Issue by sendreagheorghe** [laravel-doctrine/orm#417]
I have installed new laravel project and I've got an error when I try to migrate entities

### package.json

```
"require": {
        "php": "^7.2",  

        "fideloper/proxy": "^4.0", 

        "laravel-doctrine/orm": "1.5.*", 

        "laravel/framework": "^6.8.0", 

        "laravel/tinker": "^2.0", 

        "ext-json": "*" 
    }
```
When i try to run **php artisan doctrine:schema:create** I've got this error

` Illuminate\Contracts\Container\BindingResolutionException  : 
Target [Doctrine\Persistence\ManagerRegistry] is not instantiable.`

```
Exception trace:
  1   Illuminate\Container\Container::notInstantiable("Doctrine\Persistence\ManagerRegistry")
      /var/www/html/Health Care/HealtCare/vendor/laravel/framework/src/Illuminate/Container/Container.php:812
  2   Illuminate\Container\Container::build("Doctrine\Persistence\ManagerRegistry")
      /var/www/html/Health Care/HealtCare/vendor/laravel/framework/src/Illuminate/Container/Container.php:681
```

**Comment by bounoable** [comment]
see https://github.com/laravel-doctrine/orm/pull/415
As a temporary fix you can manually downgrade doctrine/persistence:

```json
{
    "doctrine/persistence": "1.2.*"
}
```

**Comment by Sendrea** [comment]
> ```json
> "doctrine/persistence": "1.2.*"
> ```

**Comment by eigan** [comment]
> 1.4.13 and 1.5.3 released with fix. Solved by marking doctrine/persistence 1.3 as a conflict. This will prevent composer from upgrading to this version.

See https://github.com/laravel-doctrine/orm/issues/414#issuecomment-567719199
